src/main/java/uk/gov/hmcts/reform/prl/services/c100respondentsolicitor/C100RespondentSolicitorService.java

```diff
@@ -110,8 +110,12 @@ public class C100RespondentSolicitorService {
 
     public static final String RESPONSE_SUBMITTED_LABEL = "# Response Submitted";
 
-    public static final String CONTACT_LOCAL_COURT_LABEL = "### Your response is now submitted. \n\n "
-        + "You can contact your local court at ";
+    public static final String CONTACT_LOCAL_COURT_LABEL = """
+        ### Your response is now submitted.
+
+
+        You can contact your local court at\s
+        """;
```

Comment:
why \s in the end of line 117? can we merge 117 and 118 with a space in between please?

Reply:
If there needs to be a trailing whitespace, it needs to be escaped.